pkg/ee/metering/pvc.go

```diff
@@ -52,29 +51,43 @@ func persistentVolumeClaimCreator(ctx context.Context, client ctrlruntimeclient.
 				"app": meteringToolName,
 			}
 
-			pvc.Spec.StorageClassName = pointer.StringPtr(seed.Spec.Metering.StorageClassName)
-			pvc.Spec.AccessModes = []corev1.PersistentVolumeAccessMode{
-				corev1.ReadWriteOnce,
-			}
-
-			pvcStorageSize, err := resource.ParseQuantity(seed.Spec.Metering.StorageSize)
-			if err != nil {
-				return fmt.Errorf("failed to parse value of metering pvc storage size %q: %v", "100Gi", err)
-			}
-
-			pvc.Spec.Resources = corev1.ResourceRequirements{
-				Requests: corev1.ResourceList{
-					"storage": pvcStorageSize,
-				},
+			if err := updatePVCStorageSizeAndName(pvc, seed); err != nil {
+				return fmt.Errorf("failed to update pvc storage class name or size: %v", err)
 			}
 
 			if err := client.Create(ctx, pvc); err != nil {
 				return fmt.Errorf("failed to create pvc %v for the metering tool: %v", meteringDataName, err)
 			}
+
+			return nil
 		}
 
 		return err
 	}
 
+	if err := updatePVCStorageSizeAndName(pvc, seed); err != nil {
+		return fmt.Errorf("failed to update pvc storage class name or size: %v", err)
+	}
+
+	return client.Update(ctx, pvc)
+}
+
+func updatePVCStorageSizeAndName(pvc *corev1.PersistentVolumeClaim, seed *kubermaticv1.Seed) error {
+	pvc.Spec.StorageClassName = pointer.StringPtr(seed.Spec.Metering.StorageClassName)
+	pvc.Spec.AccessModes = []corev1.PersistentVolumeAccessMode{
+		corev1.ReadWriteOnce,
+	}
+
+	pvcStorageSize, err := resource.ParseQuantity(seed.Spec.Metering.StorageSize)
+	if err != nil {
+		return fmt.Errorf("failed to parse value of metering pvc storage size %q: %v", "100Gi", err)
```

Comment:
Was it intended to include `seed.Spec.Metering.StorageSize` instead of the hard coded `"100Gi"`?

Reply:
This is actually a copy and past